src/util/writing-nft.js

```diff
@@ -22,8 +22,8 @@ export function getPowerFactor(batch) {
   return (
     1 -
     (batch - PRICE_GROWTH_POWER_DEGRADE_BATCH_BEGIN) /
-      (PRICE_GROWTH_POWER_DEGRADE_BATCH_BEGIN -
-        PRICE_GROWTH_POWER_DEGRADE_BATCH_END)
+      (PRICE_GROWTH_POWER_DEGRADE_BATCH_END -
+        PRICE_GROWTH_POWER_DEGRADE_BATCH_BEGIN)
```

Comment:
Would this function be helpful?
https://github.com/likecoin/likecoin-api-public/blob/master/src/util/api/likernft/purchase.js#L33

Reply:
should we refactor it into a lib?

Reply:
Ping @williamchong